libnopegl/src/text.c

```diff
@@ -57,17 +57,21 @@ int ngli_text_set_string(struct text *s, const char *str)
     if (ret < 0)
         goto end;
 
+    /* Convert canvas dimension from 26.6 to integer (rounded down) */
+    s->width /= 64;
+    s->height /= 64;
```

Comment:
maybe introduce and use NGLI_I26D6_TO_I32() ?

Reply:
Added `NGLI_I26D6_TO_I32_TRUNCATED()` and used it (we could have different rounding)

Reply:
Thanks